### PR TITLE
Styleguide redesign + Styles Docs

### PR DESCRIPTION
**Pull request by cmdalbem**
Merging the Styleguide page redesign branch with the new Styles + Tokens docs so we can make sure both are working well together.

Feel super welcome to contribute in any or both the initiatives!

### CODE REVIEW

File: docs/components.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## Usage
+
+### Install
+
+```sh noeditor static
+yarn add @vtex/styleguide --exact
```
Comment (klzns) [inline docs/components.md]:
```suggestion
yarn add @vtex/styleguide
```

File: docs/components.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+# or
+npm install @vtex/styleguide --save-exact
```
Comment (klzns) [inline docs/components.md]:
```suggestion
npm install @vtex/styleguide
```

File: docs/components.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+```
+
+### VTEX IO apps
```
Comment (klzns) [inline docs/components.md]: Colocar isso antes do npm

File: docs/components.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+Add the styleguide to dependencies on `manifest.json`:
+
+```sh noeditor static
+"dependencies": {
+  "vtex.styleguide": "5.x"
```
Comment (klzns) [inline docs/components.md]:
```suggestion
  "vtex.styleguide": "7.x"
```

File: docs/components.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+Importing components:
+
+```js noeditor static
+import { Button } from 'vtex.styleguide'
+```
+
+### Other projects
+
+```sh noeditor static
+yarn add @vtex/styleguide --exact
```
Comment (klzns) [inline docs/components.md]:
```suggestion
yarn add @vtex/styleguide
```

File: docs/components.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+# or
+npm install @vtex/styleguide --save-exact
```
Comment (klzns) [inline docs/components.md]:
```suggestion
npm install @vtex/styleguide
```

File: docs/styles/colors.md [hunk 7]
```diff
@@ -0,0 +1,218 @@
...
+```jsx 
+<div class="flex mv3 items-center">
+    <div class="pa3 br2 bg-action-primary hover-bg-action-primary active-bg-action-primary c-on-action-primary hover-c-on-action-primary active-c-on-action-primary dib mr3">Aa</div>
+    <div class="pa3 br2 c-action-primary hover-c-action-primary active-c-action-primary dib mr5 mv0 ba b--action-primary hover-b-action-primary active-b-action-primary">Aa</div>
+</div>
+
+```
+
+
+<!-- -->
```
Comment (kevinch) [inline docs/styles/colors.md]: Useful?

**Pull request merged**